src/lib/generators/server/generate-server.ts

```diff
@@ -1,338 +1,367 @@
-import { ImportDeclarationStructure, OptionalKind, Scope, SourceFileStructure, StructureKind, VariableDeclarationKind } from 'ts-morph';
-import { UI } from '../../interfaces/ui.model';
-import pluralize from 'pluralize';
```

Comment:
These settings are related to .editorconfig
Engage it or use a better IDE like WebStorm/PhpStorm
False positive #3

Reply:
I already use visual studio code, but there is no linter guide added to your repo, we eill add one later, so all contributors have the same code style.

Reply:
Use .editorconfig.
False positive again for 6th time.